hubble/src/arb.rs

```diff
@@ -90,40 +104,38 @@ impl Arb {
     async fn execution_height_of_beacon_slot(&self, slot: u64) -> Result<u64> {
         // read the next_node_num at l1.execution_height(beacon_slot), then from there filter for `NodeCreated`
         let next_node_num = self.next_node_num_at_beacon_slot(slot).await?;
-
         let [event] = self
             .l1_client
             .get_logs(
-                &ethers::types::Filter::new()
+                &Filter::new()
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/unionlabs/union/pull/2693#issuecomment-2293655644>#ef6b49</a><span> - 08/16/2024, 03:05 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The change from 'ethers::types::H160' to 'alloy::primitives::Address' and the use of 'FixedBytes::from_slice' may introduce issues if the address length is not validated. Alloy's 'Address' expects a fixed length, and any deviation can cause runtime errors. A similar issue was also found in hubble/src/arb.rs (line 117-118).
</div>
<div id="fix">
<b>Fix:</b> Ensure that the address length is validated before converting it to 'FixedBytes'. This can prevent potential runtime errors due to incorrect address lengths.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -113,6 +113,10 @@
                      alloy::eips::BlockNumberOrTag::Earliest..alloy::eips::BlockNumberOrTag::Latest,
                  )
                  .address(alloy::primitives::Address(
 +                    if self.rollup_finalization_config.l1_contract_address.0.len() != 20 {
 +                        return Err(eyre!("Invalid address length"));
 +                    }
                      FixedBytes::from_slice(&self.rollup_finalization_config.l1_contract_address.0),
                  ))
                  .event_signature(NodeCreated::SIGNATURE_HASH)
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged